ssc/Pos/Ssc/GState/BlockLogic.hs

```diff
@@ -0,0 +1,255 @@
+{-# LANGUAGE RankNTypes #-}
+
+module Pos.Ssc.GState.BlockLogic
```

Comment:
We usually call such modules `VAR` (verification + application + rollback).

Reply:
thanks, I will rename the module